cmake/usage/CMakeLists.txt

```diff
@@ -3,6 +3,18 @@
 string(REPLACE ";" " " BOARD_ROOT_SPACE_SEPARATED "${BOARD_ROOT}")
 string(REPLACE ";" " " SHIELD_LIST_SPACE_SEPARATED "${SHIELD_LIST}")
 
+string(REPLACE " " ";" SHIELD_LIST "${SHIELD_LIST_SPACE_SEPARATED}")
+set(sorted_shield_list ${SHIELD_LIST})
+list(SORT sorted_shield_list)
+set(sorted_shield_cmds)
+foreach(shield ${sorted_shield_list})
+  list(APPEND sorted_shield_cmds COMMAND ${CMAKE_COMMAND} -E echo "  ${shield}")
+endforeach()
+add_custom_target(shields ${sorted_shield_cmds})
```

Comment:
This is a bit short to place in a `shields.cmake` but I don't feel it belongs in this file.

So either in `shields.cmake` or alternatively as part of boards.cmake.

Reply:
> So either in `shields.cmake` or alternatively as part of boards.cmake.

OK, I will add a 'shields.cmake'.